### PR TITLE
Admin page mark/unmark mentors

### PR DESCRIPTION
**Pull request by Lbatson**
implemented feature from #4 to select and deselect mentors from admin pages. checks if user is admin before allowing access. insures mentor wont remove themselves. updates all users is_mentor field based on selection when saved.

### CODE REVIEW

**Comment by raylu** [comment]
1. this is pretty messy, please squash some commits
2. does this merge onto master (with the merged css changes?)

**Comment by Lbatson** [comment]
alright, commits squashed. fixed the multi line to single line quotes. removed the css changes i made that weren't part of the admin page. I'll get those sorted out when i go back and edit the theme again.

File: templates/admin.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+{% block css %}
+	<link rel="stylesheet" type="text/css" href="/css/users.css" />
+{% end %}
+
+{% block main %}
+	<div class="row">
+		<div class="col-md-12">
+			<form action="/admin" method="post">
+				<input class="btn btn-success pull-right" type="submit" value="Save Changes">
+				<br><br><br>
```
Comment (raylu) [inline templates/admin.html]: :(

**Comment by Lbatson** [comment]
heres a gif of the feature currently

test

File: server.py [hunk 2]
```diff
@@ -216,6 +216,35 @@ def post(self):
...
+				users.append(user['github_id'])
+			remove_mentor_set = set(users) - add_mentor_set
+			remove_mentor_set.discard(self.current_user['github_id'])
+			if len(add_mentor_set) > 0:
+				self.db.update_is_mentor(add_mentor_set, 1)
+			if len(remove_mentor_set) > 0:
+				self.db.update_is_mentor(remove_mentor_set, 0)
+			self.redirect('/admin')
+		else:
+			self.redirect('/')
```
Comment (raylu) [inline server.py]: this should definitely be `raise tornado.web.HTTPError(403)`

the other auth-ed handlers check auth, raise if conditions aren't met, and everything else is unindented

File: static/css/base.ccss [hunk 3]
```diff
@@ -87,6 +87,19 @@ body:
 		left: auto
 		right: 0
 		width: 85%
+
+/* admin.html */
```
Comment (raylu) [inline static/css/base.ccss]: :(

File: server.py (hunk 2 above)
Comment (Lbatson) [inline server.py]: updated this

File: static/css/base.ccss (hunk 3 above)
Comment (Lbatson) [inline static/css/base.ccss]: removed

**Comment by raylu** [comment]
wait, i totally didn't realize we already had a bunch of page-specific css in the same base.ccss file...

the :( was because we already have page-specific css in separate files

**Comment by Lbatson** [comment]
ok i'll split out the page sepecific stuff to their own files

**Pull request merged**